src/renderer/views/Trending/Trending.js

```diff
@@ -42,13 +52,50 @@ export default Vue.extend({
     }
   },
   mounted: function () {
-    if (this.trendingCache && this.trendingCache.length > 0) {
+    if (this.trendingCache[this.currentTab] && this.trendingCache[this.currentTab].length > 0) {
       this.shownResults = this.trendingCache
     } else {
       this.getTrendingInfo()
     }
   },
   methods: {
+    changeTab: function (tab, event) {
+      if (event instanceof KeyboardEvent) {
+        if (event.key === 'Tab') {
+          return
+        } else if (event.key === 'ArrowLeft' || event.key === 'ArrowRight') {
+          // navigate trending tabs with arrow keys
+          const index = this.tabInfoValues.indexOf(tab)
+          // tabs wrap around from leftmost to rightmost, and vice versa
+          tab = (event.key === 'ArrowLeft')
+            ? this.tabInfoValues[(index > 0 ? index : this.tabInfoValues.length) - 1]
+            : this.tabInfoValues[(index + 1) % this.tabInfoValues.length]
+
+          const tabNode = $(`#${tab}Tab`)
+          event.target.setAttribute('tabindex', '-1')
+          tabNode.attr('tabindex', '0')
+          tabNode.focus()
+        }
+
+        event.preventDefault()
+        if (event.key !== 'Enter' && event.key !== ' ') {
+          return
+        }
+      }
+      const currentTabNode = $('.channelInfoTabs > .tab[aria-selected="true"]')
+      const newTabNode = $(`#${tab}Tab`)
+
+      // switch selectability from currently focused tab to new tab
+      $('.channelInfoTabs > .tab[tabindex="0"]').attr('tabindex', '-1')
+      newTabNode.attr('tabindex', '0')
+
+      currentTabNode.attr('aria-selected', 'false')
+      newTabNode.attr('aria-selected', 'true')
+      this.currentTab = tab
+      this.getTrendingInfo()
```

Comment:
Even with my below changes fixed, this changeTab() method will not keyboard navigable because of this line. The calls to .focus() in this function are useless because the page info has to load again. You'll have to find a way to apply the focus after that loading happens. I fixed a similar problem in 558eb3f if you want to find some inspiration there.